site/documentation/content/deployment/azure.md

```diff
@@ -0,0 +1,207 @@
++++
+title = "Microsoft Azure"
+weight = 475
+aliases = [
+    "/deployment/azure"
+]
++++
+
+This guide describes how Eclipse Hono™ can be deployed on Microsoft Azure. It includes:
```

Comment:
I do not quite see the necessity for having a separate document for deployment on Azure. In fact, we have put a lot of effort in always using Helm to deploy Hono, regardless of where the Kubernetes cluster is running.

FMPOV it would therefore be preferable to include Azure specific steps to create a Kubernetes cluster either in the existing `create-kubernetes-cluster.md` document or create a `create-kubernetes-cluster-on-azure.md` document and link to it from the existing document. In any case, the instructions for deploying Hono using Helm should go into the existing `helm-based-deployment.md` document. You are welcome to add a section there which deals with the specifics of specifying the loadbalancer IPs and how to use ServiceBus instead of Artemis.
Otherwise we will end up with a separate deployment document for each specific combination of options ...

Reply:
@sophokles73 understood. Thank you for the feedback. I will try to find a way and come back to you.

Reply:
@sophokles73 done, I split the guidance into the two existing pages.